__tests__/src/selectors/canvases.test.js

```diff
@@ -400,7 +401,7 @@ describe('getVisibleCanvasNonTiledResources', () => {
   });
 
   describe('getVisibleCanvasCaptions', () => {
-    it('returns canvases resources', () => {
+    it('returns v2 canvases resources', () => {
```

Comment:
I am actually confused about this, because the `videoFixture` (from `import videoFixture from '../../fixtures/version-3/0015-start.json'`) is supposedly a IIIF v3 resource, but it has its captions in the old v2 style.